workspace/user.go

```diff
@@ -4,15 +4,22 @@ import (
 	"fmt"
 	"io"
 
+	"github.com/pkg/errors"
+
 	"github.com/astronomer/astro-cli/houston"
 	"github.com/astronomer/astro-cli/pkg/printutil"
+	"github.com/astronomer/astro-cli/user"
 )
 
 // Add a user to a workspace with specified role
-func Add(workspaceId, email, role string, client *houston.Client, out io.Writer) error {
+func Add(workspaceID string, email, role string, client *houston.Client, out io.Writer) error {
+	if !user.IsValidEmail(email) {
+		return errors.New(email + " is an invalid email address")
+	}
+
 	req := houston.Request{
 		Query:     houston.WorkspaceUserAddRequest,
-		Variables: map[string]interface{}{"workspaceId": workspaceId, "email": email, "role": role},
+		Variables: map[string]interface{}{"workspaceId": workspaceID, "email": email, "role": role},
```

Comment:
@adam2k can you pls explain this change to me? it's go style to use `workspaceId`.

Reply:
Are we using go lint?  The linter is suggesting to make this change.

![Screen Shot 2020-08-19 at 10 31 38 AM](https://user-images.githubusercontent.com/749118/90648177-34667600-e207-11ea-993e-10f0731db14c.png)

Reply:
IMO it's out of scope of this ticket. If you prefer to add `golint` it OK with me, but it should be seperate PR/issue also we need to update circleci pipeline to run it on each commit.

Reply:
so add to Makefile and to https://github.com/astronomer/astro-cli/blob/90c23704094c4bc047523c3db13a8f2868063565/.circleci/config.yml#L91 and to Dockerfile/ or dev deps.

Reply:
oh okay, I thought this is what we were using.  I can revert and ignore in my editor.  I'll make that a separate issue.